src/gan/gen_pyt/model/utils.py

```diff
@@ -0,0 +1,72 @@
+# coding=utf-8
+
+
+import math
+import numpy as np
+from torch.autograd import Variable
+
+from multivac.src.gan.gen_pyt.model.parser import Parser
+
+class GloveHelper(object):
+    def __init__(self, glove_file):
+        self.glove_file = glove_file
+        embeds = np.zeros((5000, 100), dtype='float32')
+        for i, (word, embed) in enumerate(self.embeddings):
+            if i == 5000: break
```

Comment:
why only the first 5000? does that matter in any significant way? is it fine to hard-code this or should it be variable?

Reply:
Noting this to follow up on in an upcoming new dev branch, after we get this checkpoint PR merged. 5000 is the default vocab size, set/defined in a couple places. I want to open this up to be arbitrary size, but that will require changes in multiple places to keep everything aligned.